draft-meynell-panrg-scion-research-questions.md

```diff
@@ -200,6 +201,14 @@ This draft assumes the reader is familiar with some of the fundamental concepts
 * One AS belongs to many ISDs?
 
 
+## DNS Service Binding
+
+The DNS Service Binding {{RFC9460, Section 14.3}} allows to easily specify a dedicated "SCION" Service Parameter.
```

Comment:
Keep in mind this would require a IANA action, not sure if this can be there in an IRTF draft. Nothing blocking, just something to keep in mind

Reply:
No action for now, I just pointed this out because saying anything that causes IANA actions raises some eyebrows. For now, let's leave it in the draft. If the draft gets adopted, then it will be a problem